scheduler/docs/scheduler-rest-api.adoc

```diff
@@ -515,6 +515,42 @@ To set a user's quota back to the default, an operator can retract the quota the
 $ curl -X DELETE -u: --negotiate "$cook_uri/quota?user=$user"
 ----
 
+==== User resource usage
+
+Each user's currently-running jobs have a resource footprint.
+This endpoint allows you to query a user's current aggregate resource usage.
+
+===== Get user's current resource usage
+
+`GET /usage?user=:user`
+
+.Possible response codes
+[options="header"]
+|====
+|Code | HTTP Meaning | Possible reason
+|200 | OK | Returns the user's current resource usage (JSON)
+|400 | Malformed | Returned if a user is not specified
```

Comment:
In the liberator resource, we have docs for codes (200, 401, 403) as opposed to (200, 400, 403) here. These should be consistent

Reply:
But it's consistent with the same wrong error codes being used for the `/quota` endpoint docs!
Like I said, we should switch to just using swagger. (But I fixed these codes here anyway.)